src/modules/score.js

```diff
@@ -0,0 +1,80 @@
+import { newScoreList } from './varibles.js';
+
+export const showScores = ({ user, score }) => {
+  const myelement = document.createElement('div');
+  myelement.className = 'score__item';
+  myelement.innerHTML = `
+  <i class="fa-solid fa-user-tie"></i><p class="name">${user}</p><p class="score">${score}</p>
+    `;
+
+  return myelement;
+};
+
+export const getAllMyGameScores = async () => {
+  try {
+    const rest = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/k9qDo2edCBNPllUNNxCr/scores/');
```

Comment:
- [ ] Why don't you store the API URL in a variable so that you will use the variable instead of using this  link anywhere in your code for best practice's sake :+1:

Reply:
@Owusu-Desmond,
thanks for your good and valuable feedback.
I will take in to account your nice suggestion in my next attempt.